varinfo/generate_umm_var.py

```diff
@@ -70,7 +72,7 @@ def generate_collection_umm_var(
         )
 
         # Parse the granule with VarInfo to map all variables and relations:
-        var_info = VarInfoFromNetCDF4(local_granule)
+        var_info = VarInfoFromNetCDF4(file_path=local_granule, config_file=config_file)
```

Comment:
Why did you make `file_path` a keyword argument?  I would leave it. 
```suggestion
        var_info = VarInfoFromNetCDF4(local_granule, config_file=config_file)
```
Isn't that the same?

Reply:
Did not realize I made the change. will undo the change if I did